CityOfDetroit/bloom
# Add stock photo when image isn't available

# Pull Request Template

## Issue

Addresses #283 
![image](https://user-images.githubusercontent.com/82653098/129089412-a9ad17f6-450b-4e6a-aada-7b41e1ab7b06.png)


## Description

When a listing doesn't include a property image, this creates a fallback stock photo default.

## Type of change


## How Can This Be Tested/Reviewed?

Open the partner portal and create a new listing with no photo attached.  Refresh the public site and see that a stock image has been added to the new listing.


## Checklist:
